tests/unit/s2n_connection_test.c

```diff
@@ -126,7 +126,7 @@ int main(int argc, char **argv)
      */
     {
         /* Carefully consider any increases to this number. */
-        const uint16_t max_connection_size = 4150;
+        const uint16_t max_connection_size = 4184;
```

Comment:
I've generally done round numbers so that single byte adds / removes don't impact it. So like let's go up to 4190, or 4200.